01-path-basics/102-your-first-cluster/readme.adoc

```diff
@@ -1,127 +1,96 @@
-= Create A Kubernetes Cluster Using kops
+= Create A Kubernetes Cluster Using EKS
 :toc:
 :icons:
 :linkattrs:
 :imagesdir: ../../resources/images
 
-This section will walk you through how to install a Kubernetes cluster on AWS using kops.
+This section will walk you through how to install a Kubernetes cluster on AWS using EKS.
 
-https://github.com/kubernetes/kops[kops, window="_blank"], short for Kubernetes Operations, is a set of tools for installing, operating, and deleting Kubernetes clusters. kops can also perform rolling upgrades from older versions of Kubernetes to newer ones, and manage the cluster add-ons.
+link:https://aws.amazon.com/eks/[Amazon Elastic Container Service for Kubernetes, window="_blank"] (Amazon EKS) makes it easy to deploy, manage, and scale containerized applications using Kubernetes on AWS.
 
-If you have not set up the link:../101-start-here[Cloud9 Development Environment, window="_blank"] yet, please do so before continuing.
-
-== Create a Kubernetes Cluster with kops
+Amazon EKS runs the Kubernetes management infrastructure for you across multiple AWS availability zones to eliminate a single point of failure. Amazon EKS is certified Kubernetes conformant so you can use existing tooling and plugins from partners and the Kubernetes community. Applications running on any standard Kubernetes environment are fully compatible and can be easily migrated to Amazon EKS.
```

Comment:
"management infrastructure" -> "control plane"

Not any standard Kubernetes environment, the version has to match.

Reply:
That's a quote from https://aws.amazon.com/eks/.